editions/tw5.com/tiddlers/widgets/ParametersWidget.tid

```diff
@@ -0,0 +1,61 @@
+caption: parameters
+created: 20220909111836951
+modified: 20221019081352266
+tags: Widgets
+title: ParametersWidget
+type: text/vnd.tiddlywiki
+
+<<.from-version "5.3.0">> The <<.wlink ParametersWidget>> widget is used within transcluded content to declare the parameters to be made available to the <<.wlink TranscludeWidget>> widget.
+
+There are shortcuts for common scenarios that can often make it unnecessary to use the <<.wlink ParametersWidget>> widget directly:
+
+* the [[Pragma: \parameters]]
+* the [[Pragma: \procedure]] for declaring procedure
+* the [[Pragma: \widget]] for declaring custom widgets
+
+The <<.wlink ParametersWidget>> widget must be used directly in the following situations:
+
+* When the default value of a parameter must be computed dynamically
+* When the `$depth` attribute is used to retrieve parameters from a parent transclusion (see below)
```

Comment:
This is extremely helpful telling the user when to use this feature over the alternatives. Nicely done!